python/docs/apis/tags/ReleaseManagementApi.md

```
@@ -1276,6 +1278,13 @@ Input Type | Accessed Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 str,  | str,  |  |
 
+# ToReleaseIdSchema
+
+## Model Type Info
+Input Type | Accessed Type | Description | Notes
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)